apps/public-web/src/EShopOnAbp.PublicWeb/Options/RemoteServices.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EShopOnAbp.PublicWeb.Options
```

Comment:
No need for this. See: https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Http.Client/Volo/Abp/Http/Client/AbpRemoteServiceOptions.cs